pandas/core/resample.py

```diff
@@ -825,7 +825,6 @@ def fillna(self, method, limit: int | None = None):
         """
         return self._upsample(method, limit=limit)
 
-    @doc(NDFrame.interpolate, **_shared_docs_kwargs)
```

Comment:
@mroeschke here I remove the docstring concatenation and added links to documentation, so the user can inform her/himself about various methods